### PR TITLE
fix(fxa-settings): Various UI tweaks related to recovery key

### PR DESCRIPTION
**Pull request by vpomerleau**
## Because

* The lightbulb image in account recovery key creation step 4 was offset on mobile
* The focus outline was misplaced on the back button

## This pull request

* Replace lightbulb svg
* Adjust styling for back button

## Issue that this pull request solves

Closes: #FXA-7542, #FXA-7553, #FXA-2526

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

image

image

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx [hunk 1]
```diff
@@ -31,21 +31,21 @@ export const FlowContainer = ({
     : ftlMsgResolver.getMsg('flow-container-back', 'Back');
   return (
     <div
-      className="max-w-lg mx-auto mt-6 p-10 tablet:my-10 flex flex-col items-start bg-white shadow tablet:rounded-xl"
+      className="max-w-lg mx-auto mt-6 p-10 mobileLandscape:my-10 flex flex-col items-start bg-white shadow mobileLandscape:rounded-xl"
```
Comment (pdehaan) [inline packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx:34]: Q: Why the change from `tablet:*` to `mobileLandscape:*`?
Or more importantly, do we need to modify the 46 other instances of `tablet:*` [in 28 unique files]?

```sh
git grep -Pn "tablet:(\w|\d|-)+" packages/ | wc -l # 46 matches
git grep -Pl "tablet:(\w|\d|-)+" packages/ | wc -l # 28 files
```

File: packages/fxa-settings/src/components/images/lightbulb.svg [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-<path d="M223.374 36.3736C223.374 39.812 221.166 47.4167 217.632 49.1836C217.19 57.1346 213.655 59.6241 208.356 58.9015C204.348 58.9015 201.288 58.9015 199.963 49.6253C198.278 46.5138 194.662 42.5577 192.896 39.0239C192.896 31.4613 200.793 22.6802 208.356 22.6802C215.918 22.6802 223.374 28.8109 223.374 36.3736Z" fill="#7542E5"/>
+<svg width="68" height="68" viewBox="0 0 68 68" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (pdehaan) [inline packages/fxa-settings/src/components/images/lightbulb.svg:1]: Unrelated nit, but the 👁️  see what the 👁️  see&hellip;
Can/should we rename this file to "graphic_lightbulb.svg" to be consistent with the other SVGs in the [/fxa-settings/src/components/images/](https://github.com/mozilla/fxa/tree/main/packages/fxa-settings/src/components/images) folder?

Also, just for giggles confirmed the other sizes in the wizard:
- [graphic_key.svg](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/images/graphic_key.svg): `<svg width="71" height="39" …>`
- [graphic_lock.svg](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/images/graphic_lock.svg): `<svg width="53" height="65" …>`
- [graphic_security_shield.svg](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/images/graphic_security_shield.svg) `<svg width="250" height="200" …>`
- [lightbulb.svg](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/images/lightbulb.svg): `<svg width="68" height="68" viewBox="0 0 68 68" …>` (I think was the only non-minimized one as well)

File: packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx (hunk 1 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx:34]: `tablet` is a bit of an outlier compared to our other media query configs - it has an upper and lower bound, compared to others like `mobileLandscape` and `desktop` which only have a minimum. In this case we'd want to styling to apply for anything larger than mobile portrait, so `mobileLandscape` is the better choice.

I'm uncertain if other uses of `tablet` intentionally targeted that specific format, or if it was applied with the intention of applying the styling to anything "tablet and larger".

File: packages/fxa-settings/src/components/images/lightbulb.svg (hunk 2 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/components/images/lightbulb.svg:1]: Good idea about the naming, and thanks for catching that it is the only non-minimized one!

File: packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx [hunk 3]
```diff
@@ -31,21 +31,21 @@ export const FlowContainer = ({
...
           onClick={onBackButtonClick}
           data-testid="flow-container-back-btn"
           title={backButtonTitle}
-          className="relative w-8 h-8 -ms-2 me-2 tablet:me-10 tablet:-ms-18"
+          className="me-4 mobileLandscape:absolute mobileLandscape:p-4 mobileLandscape:-start-24"
         >
-          <BackArrow className="w-8 h-auto pe-2 absolute top-1/2 start-1/4 transform -translate-y-1/2 ltr:-translate-x-1/2 rtl:translate-x-1/2 rtl:-scale-x-100 fill-current" />
+          <BackArrow className="w-6 h-auto fill-grey-400 rtl:transform rtl:-scale-x-100" />
         </button>
-        <h1 className="font-header text-lg mb-1">{title}</h1>
```
Comment (LZoog) [inline packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx:48]: Was removing `mb-1` intentional? I'm wondering if that affects any other layouts using this, or if our other headers had a `margin` anyway that was collapsed.

File: packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx [hunk 4]
```diff
@@ -31,21 +31,21 @@ export const FlowContainer = ({
...
       <Head title={title} />
 
-      <div className="flex items-center">
+      <div className="relative flex items-center">
         <button
           onClick={onBackButtonClick}
           data-testid="flow-container-back-btn"
           title={backButtonTitle}
-          className="relative w-8 h-8 -ms-2 me-2 tablet:me-10 tablet:-ms-18"
+          className="me-4 mobileLandscape:absolute mobileLandscape:p-4 mobileLandscape:-start-24"
```
Comment (LZoog) [inline packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx:44]: I think you can take `me-4` off as well. 😏 

With these changes I think the parent div `flex items-center` just scoots the arrow down slightly. Not sure if it'd be nicer to take those off and have more obvious control of the arrow with `top` in case we need to tweak later, or leave as-is. Nice improvement either way though, you'll be closing https://mozilla-hub.atlassian.net/browse/FXA-2526 inadvertently here too!

**Review by LZoog — APPROVED** [review]
👍 Just have one question about the margin-bottom removal. Thanks for all these fixes 🎉

File: packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx (hunk 3 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx:48]: It was intentional, but it would make sense to keep the margin if/when there is a sub-heading. I'll move the margin to the h2 element instead.

File: packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx (hunk 4 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx:44]: The spacing is needed for small screens, where we only want spacing between the arrow and the heading. `me-4` adds spacing either to the left or right of the button depending on LTR/RTL
